app/views/questions/_questions.html.erb

```diff
@@ -2,15 +2,7 @@
 <div class="card-view questions-page row">
 	<% questions.each_with_index do |node,i| %>
 		<div class="<% if @widget %>col-xs-4 col-sm-4 col-md-4 <% end %>col-lg-4" style="margin-top:20px;">
-      <div class="card bg-white rounded<% if node.status == 4 %> moderated<% end %>">
-        <div class="card-header">
-          <%= render partial: 'dashboard/node_moderate', locals: { node: node } %>
-          <h4><a <% if @widget %>target="_blank"<% end %> href="<%= node.path(:question) %>"><%= node.title %></a></h4>
-        </div>
-        <div class="card-footer">
-          <p class="meta"><%= render partial: "dashboard/node_meta", locals: { node: node } %></p>
-        </div>
-      </div>
+      <%= render partial: 'notes/card', locals: { node: node, i: i, tagname: tagname } %>
```

Comment:
This looks great. What do you think might be happening with the tests? The instance variable `@widget` should be persisted into the partial template. What are the logs saying? 

Thanks!!

Reply:
I'm new here, first day, but I did see an error on the test run that may help.
undefined local variable or method `tagname' for #<#<Class:0x000055de006c8e30>:0x000055de0460da78>
            app/views/questions/_questions.html.erb:5:in `block in _app_views_questions__questions_html_erb__4321177480707985642_47206022398320'
            app/views/questions/_questions.html.erb:3:in `each_with_index'
            app/views/questions/_questions.html.erb:3:in `_app_views_questions__questions_html_erb__4321177480707985642_47206022398320'
            app/views/questions/index.html.erb:79:in `_app_views_questions_index_html_erb__3421750489030126837_47206022279460'
            test/integration/login_flow_test.rb:58:in `block in <class:LoginFlowTest>'